src/components/download-track-progress-drawer/DownloadTrackProgressDrawer.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback } from 'react'
+
+import LottieView from 'lottie-react-native'
+import { Platform, StyleSheet, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import Drawer from 'app/components/drawer'
+import Text from 'app/components/text'
+import { useDrawer } from 'app/hooks/useDrawer'
+import {
+  getDownloadedPercentage,
+  getFileName,
+  getFetchTask
+} from 'app/store/download/selectors'
+
+const styles = StyleSheet.create({
+  view: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    paddingTop: 16,
+    paddingHorizontal: 24
+  },
+
+  title: {
+    fontFamily: 'AvenirNextLTPro-Bold',
+    fontSize: 21,
```

Comment:
Are these colors all the same for dark mode? I would recommend using the `useThemedStyles` hook to get access to the theme colors